docs/developer-center/new-guides/quickstart.md

```diff
@@ -0,0 +1,75 @@
+## Quickstart
+
+Hi! Glad to see you around! This guide walks you through the common steps a data scientist has to follow in their spatial analyses and how CARTOframes can help in each of them. We will follow a real use case to show it. We encourage you to have your environment ready so you can learn how to use the library by using it. If you haven't done it yet, check this guide.
+
+### Real use case
+
+Imagine that you are a data scientist at a bikesharing company and you want to know how well the company has placed their stations. Apart from that, you would also like to recommend new spots so the company can continue growing. Before starting, you list the steps you have to follow to achieve it:
+
+- Get and explore company's data
+- Discover and get meaninful data
+- Analyse which are the best and worst spots
+- Share the results with your team
+
+Let's start!
+
+
+### Get and explore company's data
+
+In order to start your work, you ask your team where you can find data about the bike stations. A team mate sends you the following file: TODO. This file contains information about the bike stations at Arlington and you check it using a pandas dataframe.
```

Comment:
This is an [example of the dataset](#) you have to start your exploration with your company data. It contains information about the bike stations at Arlington and you check it using a pandas dataframe.